README.md

```diff
@@ -65,7 +66,7 @@ the install instructions for Go](http://golang.org/doc/install.html).
 
 To install cli, simply run:
 ```
-$ go get github.com/urfave/cli
```

Comment:
⭐⭐ _(seriously consider)_ With respect to changes made to this file, I would like it if we split the docs into `v1` and `v2` docs. I've started that here => https://github.com/urfave/cli/pull/896

Reply:
How do we want to take this up?